### PR TITLE
basic support for latest activity

### PR DESCRIPTION
**Pull request by lowellbander**
image

### CODE REVIEW

**Comment by lowellbander** [comment]
thank you so much @bluquar for pair programming with me on this PR! I'm really looking forward to being a more capable coder on this project!

File: src/server/objects/aid_request/object_fields/latestEvent.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+> = {
+  resolve: async (
+    { _id }: Document<string, unknown, AidRequestType>,
+    _args: Record<string, never>,
+    req: Express.Request,
+  ): Promise<string> => {
+    assertLoggedIn(req, 'latestEvent');
+    const aidRequest = await AidRequestModel.findById(_id);
+    if (aidRequest == null) {
+      throw new Error('No request found for this ID');
```
Comment (lowellbander) [inline src/server/objects/aid_request/object_fields/latestEvent.ts:29]: #94

File: src/server/objects/aid_request/object_fields/latestEvent.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    }
+    if (aidRequest.history.length === 0) {
+      return 'No activity (yet!)';
```
Comment (lowellbander) [inline src/server/objects/aid_request/object_fields/latestEvent.ts:32]: next up I think I'll have this reference the date the request was recorded and by whom

**Comment by bluquar** [comment]
Looks good to me :) A couple next steps:

1. Show "created by __" if there is no history
2. Only show the latest "Add" event that doesn't have a later corresponding "Remove" event.  To do this you could do something like write a function that takes the history array and removes all "Add"/"Remove" pairs, and return the latest remaining element.

**Review by bluquar — APPROVED** [review]

**Pull request merged**